CONTRIBUTING.md

```diff
@@ -0,0 +1,159 @@
+3. Ensure the test and linting suite passes (`npm run prepush`).
+Follow the [JavaScript](https://github.com/airbnb/javascript) styleguide from Airbnb with the [lisk extension](https://github.com/LiskHQ/eslint-config-lisk-base).
+4. Submit a pull request via GitHub. Include issue numbers in the PR title, at the end with: ```Description - Closes #IssueNumber```.
+5. Check that Jenkins CI tests pass (pull request turns green).
+First time contributors will need to wait for a trusted team member to start Jenkins CI on a Pull Request.
+
+### Reporting Bugs
+
+This section guides you through submitting a bug report for Lisk-JS. Following these guidelines helps maintainers and the community understand your report :pencil:, reproduce the behavior :computer: :computer:, and find related reports :mag_right:.
+
+Before creating bug reports, please check [this list](#before-submitting-a-bug-report) as you might find out that you don't need to create one. When you are creating a bug report, please [include as many details as possible](#how-do-i-submit-a-good-bug-report). Fill out [the required template](ISSUE_TEMPLATE.md), the information it asks for helps us resolve issues faster.
+
+> **Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+
+#### Before Submitting A Bug Report
+
+* **Check the [FAQs](https://docs.lisk.io/docs/faq)** for a list of common questions and problems.
+* **Determine [which repository the problem should be reported in](https://github.com/LiskHQ)**.
+* **Perform a [cursory search](https://github.com/search?utf8=%E2%9C%93&q=+is%3Aissue+org%3ALiskHQ&type=)** to see if the problem has already been reported. If it has **and the issue is still open**, add a comment to the existing issue instead of opening a new one.
+
+#### How Do I Submit A (Good) Bug Report?
+
+Bugs are tracked as [GitHub issues](https://guides.github.com/features/issues/). After you've determined [which repository](https://github.com/LiskHQ) your bug is related to, create an issue on that repository and provide the following information by filling in [the template](ISSUE_TEMPLATE.md).
+
+Explain the problem and include additional details to help maintainers reproduce the problem:
+
+* **Use a clear and descriptive title** for the issue to identify the problem.
+* **Describe the exact steps which reproduce the problem** in as many details as possible. For example, start by explaining how you started Lisk-JS, e.g. with NodeJS or in the Browser, or how you started Lisk-JS otherwise. When listing steps, **don't just say what you did, but explain how you did it**. For example, if you have used an API model provide the configuration you have chosen and the functions you have executed. **Make sure to erase sensitive information from the configuration or details you are passing - NEVER SHARE YOUR SECRET PASSPHRASES OR PRIVATE KEYS**.
```

Comment:
"with Node.js or in a browser (which one? which version?)"

Reply:
The bit about configuration overlaps with the section below on configuration and environment.

Reply:
Yes, thanks. Configuration and environment are all mentioned in this block, the extra block for it has been taken out.